decidim-core/app/cells/decidim/wizard_step_form/wizard_header.erb

```diff
@@ -9,10 +9,10 @@
   <%= wizard_header_similar_resources_count %>
 </h2>
 
-<div class="hide-for-large m-bottom">
+<div class="hide-for-large">
```

Comment:
The `.hide-for-large` class is [defined for emails](https://github.com/decidim/decidim/blob/a8497d07e6af3cc362178a1588db39b1d180d5a4/decidim-core/app/packs/stylesheets/decidim/legacy/email.scss#L952) and this is not used in the emails as far as I see  
```suggestion
<div>
```